code/modules/client/_client_data.dm

```diff
@@ -0,0 +1,75 @@
+// client is deleted upon logout, but we want to keep some data
+/datum/client_data
```

Comment:
Rename to match TG

https://github.com/tgstation/tgstation/blob/master/code%2Fmodules%2Fclient%2Fplayer_details.dm